src/Tools/dotnet-watch/test/Scenario/WatchableApp.cs

```diff
@@ -104,12 +104,15 @@ public void Start(IEnumerable<string> arguments, [CallerMemberName] string name
                 WorkingDirectory = SourceDirectory,
                 EnvironmentVariables =
                 {
-                    ["DOTNET_CLI_CONTEXT_VERBOSE"] = bool.TrueString,
```

Comment:
FYI @JunTaoLuo, made this change after you approved. It fixes some flakiness with the tests due to https://github.com/dotnet/sdk/issues/10688